exercises/02-publish-subscribe/README.md

```diff
@@ -369,7 +369,13 @@ Note: If ServicePulse doesn't seem to be running, or it cannot connect to Servic
 
 ### Step 5
 
-Let's install the ServiceControl [heartbeat plugin](https://docs.particular.net/servicecontrol/plugins/heartbeat) into the NServiceBus endpoints. Install this plugin into every project that hosts an endpoint via the Visual Studio NuGet user interface or via the Package Manager Console.
+Let's install the ServiceControl [heartbeat plugin](https://docs.particular.net/servicecontrol/plugins/heartbeat) into the NServiceBus endpoints. 
```

Comment:
The URL should be changed to https://docs.particular.net/monitoring/heartbeats/install-plugin?version=heartbeats_2